src/ModalContent/ModalContent.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { dcnb } from "cnbuilder";
+import PropTypes from "prop-types";
+import { DismissButton } from "../DismissButton/DismissButton";
+import { Button } from "../Button/Button";
+
+/**
+ * ModalContent Component
+ *
+ * HTML ModalContent element
+ */
+
+export const ModalContent = ({
+  className,
+  onClose,
+  children,
+  actionButton,
+  footerChildren,
+  cancelButtonProps,
+  ...props
+}) => (
+  <div
+    className={dcnb(
+      "su-bg-white su-max-w-4xl lg:su-max-w-5xl su-flex su-flex-col su-rs-pt-2 su-rs-pr-3 su-rs-pb-4 su-rs-pl-5",
+      className
+    )}
+    {...props}
+  >
+    <DismissButton
+      id="closeModal"
+      icon="x"
+      className="su-ml-auto su-rs-mb-2 su-text-digital-blue hocus:su-text-black"
+      onClick={onClose}
+      iconProps={{
+        className: "su-ml-02em",
+      }}
+      srText="Close"
+      text="Dismiss"
+    />
+    <div className="su-rs-mb-3">{children}</div>
+    <div className="su-flex su-justify-center su-rs-mb-2">
+      <Button onClick={onClose} {...cancelButtonProps}>
+        Go Back
+      </Button>
+      {actionButton}
+    </div>
```

Comment:
@Lyafel I imagine that we'll need the option to add any custom buttons in the CTA area so we probably wouldn't want to hard code a "Go Back" button.

Perhaps take a look at the Card component. I can see us having these subcomponents (HeadlessUI style, also like the Card component). All of these are optional:
- ModalContent.Title
- ModalContent.Body
- ModalContent.Cta (user can pass in one or multiple Cta Buttons, or none)
- ModalContent.Footer